utilities/spmup_setparallel.m

```diff
@@ -22,7 +22,7 @@ function spmup_setparallel(varargin)
 end
 
 addons = ver;
-if any(strcmpi('Parallel Computing Toolbox',arrayfun(@(x) x.Name, addons, "UniformOutput",false)))
```

Comment:
the reason I did not use ismember is, if I remember, name differ slightly per matlab version and somehow did not always work (nor 100% sure)

Reply:
ok I thought something like that was going on 

but this arrayfun fails on Matlab 2018a so I was trying to get around it

Reply:
:-( same issue as with fileparts I bet, since 2019 all the any all contains etc.. can take cellarrays for comparing
maybe `tmp = arrayfun(@(x) x.Name, addons, "UniformOutput",false)` and then see how to use `tmp`

Reply:
trying this
https://www.mathworks.com/matlabcentral/answers/222647-test-if-toolbox-is-installed#answer_564028